pkg/envtest/server.go

```diff
@@ -58,8 +69,9 @@ func (te *Environment) Stop() error {
 // Start starts a local Kubernetes server and updates te.ApiserverPort with the port it is listening on
 func (te *Environment) Start() (*rest.Config, error) {
 	te.ControlPlane = integration.ControlPlane{}
+	te.ControlPlane.APIServer = &integration.APIServer{Args: defaultKubeAPIServerFlags}
```

Comment:
This is fine for now.  Would you create an issue to allow users more fine grain control of the flags passed to the apiserver?

Reply:
Done: https://github.com/kubernetes-sigs/testing_frameworks/issues/55